subprojects/docs/src/docs/userguide/running-builds/performance.adoc

```diff
@@ -1,11 +1,13 @@
 [[performance_gradle]]
 = Improve the Performance of Gradle Builds
 
-Build performance is critical to productivity.
-The longer builds take to complete, the more likely they'll disrupt your development flow.
-Builds run many times a day, so even small waiting periods add up.
-The same is true for Continuous Integration (CI) builds: the less time they take, the faster you can react to new issues
-and the more often you can experiment.
+TIP: Want faster Gradle Builds? https://gradle.com/training/#build-cache-deep-dive[Register here] for our Build Cache training session to learn how Gradle Enterprise can speed up builds by up to 90%.
+
+Build performance is critical to your productivity.
+The longer the build takes to complete, the more likely you'll be taken out of your development flow.
+On top of that, since you run the build many times a day, even small periods of waiting can add up to significant disruption.
+The same is true for builds run on CI: the faster they are, the faster you can react to new issues and the more capacity you will have to do innovative experiments.
+
```

Comment:
It looks like you've inadvertently brought an old paragraph of prose into the updated performance page. This tip now exists 12 lines below this new tip, and this paragraph has been replaced by a different paragraph. Could you restore this to the recently updated state and instead update the tip on line 16?

Reply:
thanks that was a merge conflict I resolved the wrong way ;)